contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -208,6 +213,34 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         return streamingFee.add(profitFee);
     }
 
+    /*
+     * Validate then set new streaming fee.
+     *
+     * @param  _newFeePercentage       Fee type and new streaming fee encoded in bytes
+     */
+    function adjustFee(
+        bytes calldata _newFeePercentage
+    )
+        external
```

Comment:
We need to validate that only the RBSet can call this function

Reply:
Why? The only state that will update is the callers, which would be the RebalancingSetToken.

Reply:
hmm i see, nevermind then